dll/win32/browseui/shellbrowser.cpp

```diff
@@ -821,6 +828,48 @@ HRESULT CShellBrowser::ApplyBrowserDefaultFolderSettings(IShellView *pvs)
     return hr;
 }
 
+UINT CShellBrowser::GetNewBrowserFlag(UINT SBSP)
```

Comment:
Don't use all-caps for a variable/parameter name. All-caps are reserved for types or defines. Use instead something like `smsbFlags` or similar.
And, what does "sbsp" mean?

(And same comment for the new two functions)

Reply:
SB means ShellBrowser, I don't know what SP means. They are the flags for the BrowseObject method.